src/libcore/macros.rs

```diff
@@ -8,6 +8,16 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+#[macro_export]
+// This stability attribute is totally useless.
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Maybe wishful thinking, but if we ever fix macro stability checking it'd be good to have accurate info here. Also for rustdoc purposes.